src/Examples/Flowsheets/HDA_flowsheet_with_distillation_solution_testing.ipynb

```diff
@@ -0,0 +1,1773 @@
+    "        unit.initialize(outlvl=idaeslog.INFO)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are now ready to initialize our flowsheet in a sequential mode. Note that we specifically set the iteration limit to be 5 as we are trying to use this tool only to get a good set of initial values such that IPOPT can then take over and solve this flowsheet for us. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "seq.run(m, function)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "<b>Inline Exercise:</b>\n",
+    "We have now initialized the flowsheet. Let us run the flowsheet in a simulation mode to look at the results. \n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the solver object\n",
+    "solver = SolverFactory('ipopt')\n",
+    "solver.options = {'tol': 1e-6, 'max_iter': 5000}\n",
+    "\n",
+    "# Solve the model\n",
+    "results = solver.solve(m, tee=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Check solver solve status\n",
+    "from pyomo.environ import TerminationCondition\n",
+    "assert results.solver.termination_condition == TerminationCondition.optimal"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Add distillation column \n",
+    "\n",
+    "This completes the simulation of the flowsheet constructed earlier. We will now \n",
+    "- Add the distillation column \n",
+    "- Connect it to the heater \n",
+    "- Add the necessary equality constraints\n",
+    "- Propogate the state variable information from the outlet of the heater to the inlet of the distillation column \n",
+    "- Fix the degrees of freedom of the distillation block (reflux ratio, boilup ratio, and condenser pressure)\n",
+    "- Initialize the distillation block.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Add distillation column to the flowsheet\n",
+    "m.fs.D101 = TrayColumn(default={\n",
+    "                        \"number_of_trays\": 10,\n",
+    "                        \"feed_tray_location\": 5,\n",
+    "                        \"condenser_type\":\n",
+    "                            CondenserType.totalCondenser,\n",
+    "                        \"condenser_temperature_spec\":\n",
+    "                            TemperatureSpec.atBubblePoint,\n",
+    "                        \"property_package\": m.fs.bt_properties,\n",
+    "                        \"has_heat_transfer\": False,\n",
+    "                        \"has_pressure_change\": False})\n",
```

Comment:
This is not something for this PR, but I see a lot of Pyomo warnings about implicit replacement of components. We should address that in the main distillation code.

Reply:
Yeah, Jaffer took a note of it. He said that he would take a look at the distillation code and figure out what the issue is.